### ISSUE TITLE
PowerShell 7.2.0 

### ISSUE THREAD

**Issue by Spratumn** [PowerShell/PSReadLine#2962]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 94
BufferHeight: 22

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

![微信截图_20211109112210](https://user-images.githubusercontent.com/32872875/140856282-2a35532f-9c8f-4c33-880a-81ff540a80c7.png)


### Environment data

```console
none
```

### Steps to reproduce

start power shell

### Expected behavior

no error

### Actual behavior

Oops, something went wrong.

**Comment by iSazonov** [comment]
Please update to latest 2.2.0-Beta4 version.

**Comment by LittleNewton** [comment]
You should use this command to upgrade PSReadLine to version **2.2.0-Beta4** before you upgrading PowerShell.

```
Update-Module -Name PSReadLine -AllowPreRelease -Force
```

**Comment by buzzfrog** [comment]
If you get this error when doing Update-Module:
Update-Module: Module 'PSReadLine' was not installed by using Install-Module, so it cannot be updated.

Try this instead:
Install-Module PSReadLine -AllowPrerelease -Force

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
